### PR TITLE
Create basic website

### LINKED ISSUE #1: Create a basic website

### PR DESCRIPTION
**Pull request by romw314**
Created the basic website. The itch.io site does not exist, but I guessed the URL, @iuevix. The name `iuevix` is free on itch.io and I guess that the game slug on itch.io will be `pay-and-wash`.

close #1

### CODE REVIEW

**Comment by romw314** [comment]
@iuevix please use [itch.io](https://romw314.itch.io) before merging this. There is also an unresolved conversation on Vercel about this.

File: styles/_values.scss
Comment (romw314) [inline styles/_values.scss:1]: @iuevix you can easily edit values here, for example change any colors.

File: styles/_values.scss [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+$button_color: red;
+$button_border_color: darkred;
+$button_hover_color: lightblue;
+$button_hover_border_color: blue;
+$button_active_color: $button_hover_color;
+$button_border_width: 10px;
+$button_active_border_width: 3px;
+$button_width: 100px;
+$button_height: 50px;
+$button_text_size: 0.6em;
```
Comment (romw314) [inline styles/_values.scss:32]: The button text should be bigger.

Comment (romw314) [reply styles/_values.scss:32]: This is not needed when I used itch.io.

**Review by iuevix — APPROVED** [review]

**Pull request merged**